shared/xccdf/services/base.xml

```diff
@@ -117,6 +117,7 @@ multiple processors.
 <ocil><systemd-check-macro enable="false" service="irqbalance" /></ocil>
 <rationale>In an environment with multiple processors (now common), the irqbalance service
 provides potential speedups for handling interrupt requests.</rationale>
+<platform idref="cpe:/o:virtualization:baremetal"/>
```

Comment:
I don't particularly like this use of platform.  From the subject of this RR, I was hoping to see some indication that would allow a rule to be specified as applicable only on bare metal, regardless of the value of platform.  Is there some other way to accomplish this ?

Background: this is something that I know we'll need to worry about in Wind River Linux at some time in the near future.